pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/TenantsBase.java

```diff
@@ -282,4 +284,17 @@ private CompletableFuture<Void> validateClustersAsync(TenantInfo info) {
             }
         });
     }
+
+    private CompletableFuture<Void> validateAdminRoleAsync(TenantInfoImpl info) {
+        // empty adminRole shouldn't be allowed
+        if (!info.getAdminRoles().isEmpty()) {
+            for (String adminRole : info.getAdminRoles()) {
+                if (StringUtils.isBlank(adminRole) || StringUtils.containsWhitespace(adminRole)) {
```

Comment:
I think white space should be allowed as long as it's not in the beginning or end. One way to check would be to validate this way:

```suggestion
                if (!StringUtils.trim(adminRole).equals(adminRole)) {
```